<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Odoo Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="update_list_button">
        <property name="text">
         <string>Update List</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QListView" name="modules_list"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="install_button">
        <property name="text">
         <string>Install</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="uninstall_button">
        <property name="text">
         <string>Uninstall</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="upgrade_button">
        <property name="text">
         <string>Upgrade</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>463</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>modules_list</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
